bedrock/firefox/templates/firefox/features/adblocker.html

```diff
@@ -24,133 +24,153 @@
-        <p>{{ _('First, they block communication from ad servers or others. Ads simply can’t get through.') }}</p>
-        <p>{{ _('Second, they hide parts of a web page that could be ads-- any section of a site that’s labeled "ad," for example. Most ad blockers use public and up-to-date lists of known ad servers.') }}</p>
-      </div>
-      <div>
-        <h3>{{ _('The Firefox ad blocker') }}</h3>
-        <p>
-          {% trans url='https://blog.mozilla.org/firefox/tracking-protection-always-on/' %}
-          Firefox doesn’t include ad blocking out of the box, but it does include <a href="{{ url }}">Tracking Protection</a>, which stops sites from installing those tracking cookies we talked about.
-          {% endtrans %}
-        </p>
-        <p>
-          {% trans url='https://support.mozilla.org/en-US/kb/tracking-protection' %}
-          It’s <a href="{{ url }}">easy to turn on</a>, and has the added benefit of making Firefox significantly faster.
-          {% endtrans %}
-        </p>
-      </div>
-    </div>
+    <h2 class="section-title">{{_('Create an tracker-free zone with Content Blocking') }}</h2>
+    <p>
+      {% trans privacy='https://restoreprivacy.com/firefox-privacy/', blocking='https://support.mozilla.org/en-US/kb/content-blocking' %}
+      On Firefox, you can use <a href="{{ privacy }}">Privacy</a> or <a href="{{ blocking }}">Content Blocking</a> settings to get even more control over ad trackers that serve you the ads.
+      {% endtrans %}
+    </p>
+  </section>
+
+  <section class="mzp-l-content l-content-image">
+    {{ high_res_img('firefox/features/adblocker/content-blocking-title.png', {'alt': '', 'class': 'mzp-c-billboard-image'}) }}
   </section>
 
   <section class="mzp-l-content l-content-narrow">
-    <h2 class="section-title">{{ _('Why use ad blockers?') }}</h2>
-    <p>{{ _('Ad blockers can also be used to make the web faster.') }}</p>
-    <p>{{ _('Ads can slow down sites while they load, or get in the way of a streamlined browsing experience.') }}</p>
-    <p>{{ _('Ads can also be flashy and annoying, distracting you from the stuff you really want to see.') }}</p>
-    <blockquote>{{ _('“Ads can slow down sites… or get in the way of a streamlined browsing experience.”') }}</blockquote>
+    <h2 class="section-title">{{_('Choose your level of protection') }}</h2>
     <p>
-    {% trans %}
-    Finally, ad companies can track your movements online. It’s called "re-targeting" and it works like this: You visit a site and click on a product. That site puts a small piece of code in your browser called a cookie. When you surf the web, sites will know you clicked on that product and they’ll display ads for it.
-    {% endtrans %}
+      {% trans %}
+      To start, click on the Firefox menu in the top right-hand corner of your screen. It looks like four lines stacked on top of each other. In the drop-down menu, click on Content Blocking. You should see a blue pop-up with different selections.
```

Comment:
I'm not sure I would call this a "pop up" since it actually opens a whole new page in a new tab, but maybe I'm splitting hairs.

Reply:
🤷‍♀️ I'm not sure what would be better?

Reply:
"panel"?

Reply:
I don't really think it's a panel either. Maybe we should ask Aleks? Personally, I think describing it as a new tab would work.